tools/alara_output_processing/alara_output_plotting.py

```diff
@@ -341,4 +571,196 @@ def plot_single_response(
     ax.grid(True)
     plt.tight_layout(rect=[0, 0, 0.85, 1])
     plt.show()
-    
\ No newline at end of file
+
+def single_time_pie_chart(
+        adf,
+        run_lbl,
+        variable,
+        threshold,
+        time_idx,
+        time_unit='s'
+    ):
+    '''
+    Create a pie chart depicting the breakdown of nuclides contributing to a
+        given response at a given cooling time.
+
+    Arguments:
+        adf (alara_output_processing.ALARADFrame): ALARADFrame containing 
+            response data from one or more ALARA runs.
+        run_lbl (str): Distinguisher of the specified ALARA run.
+        variable (str): Name of the response variable.
+        threshold (float): Proportional threshold for small-value aggregation.
+        time_idx (int): Cooling time interval number (i.e. 0 for shutdown).
+        time_unit (str, optional): Optional paramter to set units for cooling
+            times. Accepted values: 's', 'm', 'h', 'd', 'w', 'y', 'c'.
+            (Defaults to 's')
+    
+    Returns:
+        None
```

Comment:
Should we return the `plt` so that the user can continue to manipulate it? and then maybe let them `show()` it when ready (separation of concerns)